onebusaway-android/src/androidTest/java/org/onebusaway/android/nav/test/NavigationTest.java

```diff
@@ -808,7 +809,14 @@ void runSimulation(int expectedGetReadyIndex, int expectedPullCordIndex) {
                 } catch (Exception e) {
                     Log.e(TAG, e.getMessage());
                 }
-                provider.locationUpdated(l);
+
+                // Code added to check for duplicate locations in the .csv log files
+                if (prevLocation == null) {
```

Comment:
Could you please collapse this IF statement to:

`if (prevLocation == null || !LocationUtils.isDuplicate(prevLocation, l)) {`